Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -2811,6 +2804,49 @@ class PropertyWrapperBaselineShift final : public PropertyWrapper<SVGLengthValue
 };
 DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(PropertyWrapperBaselineShift);
 
+DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(PropertyWrapperTextUnderlineOffset);
```

Comment:
I wish I hadn't made such fine-grained heap identifiers for every class in Animation. Might be worth a follow-up patch to make a generic "Animation" heap allocator then every Animation class can just use `WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER(Animation)`.

If you don't want to do that, that's fine too I can do it some other time.

Reply:
Makes sense, yeah I'd love to pick it up. I can submit a patch for it after this one.